### DISCUSSION TITLE
How could I change dialog backgroud color

### DISCUSSION THREAD

**Discussion by dary123** [mono0926/adaptive_dialog#107] [Q&A] [answered]
How could I change dialog backgroud color

**Comment by mono0926** [comment, marked as answer]
@dary123 

Wrap with Theme like this:
https://github.com/mono0926/adaptive_dialog/blob/e24c51c6ce6454291a085c96c88e573de97d4eff/example/lib/pages/alert_page.dart#L99-L100

If you need more flexibility, please implement it yourself without this package.
